### PR TITLE
[pgadmin4] Improve examples

### PR DESCRIPTION
**Pull request by MaxymVlasov**
#### What this PR does / why we need it:

Some part of configuration not fully documented or have mistakes which greatly complicates the work with the chart for newbies. This PR try fix part of this.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped - **generally, not needed, but without it tests failed**
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/pgadmin4/values.yaml [hunk 1]
```diff
@@ -130,10 +138,11 @@ existingSecret: ""
 
 ## pgAdmin4 startup configuration
 ## Values in here get injected as environment variables
-##
+## Needed chart reinstall for apply changes
 env:
-  email: chart@example.local
+  email: chart@example.local  # can be email or nickname
```
Comment (rowanruseler) [inline charts/pgadmin4/values.yaml:143]: Could you place this comment above rather then behind?

File: charts/pgadmin4/README.md [hunk 2]
```diff
@@ -98,6 +98,6 @@ provided while installing the chart. For example:
 $ helm install runix/pgadmin4 --name my-release -f values.yaml
 ```
 
-> **Tip**: You can use the default [values.yaml](values.yaml)
+> **Tip**: You can use the default [values.yaml](https://github.com/rowanruseler/helm-charts/blob/master/charts/pgadmin4/values.yaml) and look on [examples](https://github.com/rowanruseler/helm-charts/blob/master/charts/pgadmin4/examples/).
```
Comment (rowanruseler) [inline charts/pgadmin4/README.md:101]: Thanks! I keep on forgetting to do this one

**Comment by rowanruseler** [comment]
@MaxymVlasov Thanks for your PR, just a small request - beside that it all looks good!

File: charts/pgadmin4/values.yaml (hunk 1 above)
Comment (MaxymVlasov) [reply charts/pgadmin4/values.yaml]: Sure, fixed

**Pull request merged**